pkg/api/handlers/libpod/containers.go

```diff
@@ -148,6 +148,11 @@ func GetContainer(w http.ResponseWriter, r *http.Request) {
 func WaitContainer(w http.ResponseWriter, r *http.Request) {
 	exitCode, err := utils.WaitContainer(w, r)
 	if err != nil {
+		if errors.Cause(err) == define.ErrNoSuchCtr {
+			utils.ContainerNotFound(w, utils.GetName(r), err)
```

Comment:
Hm. Should we error here? My concern is `--rm` - we could race against cleanup and error on a container being removed.

Reply:
So just log it?

Reply:
Yeah, logging seems right. Maybe someone will complain in the future and we'll have to do better, but for now this seems safest